mongodb-storage/src/main/java/io/lumeer/mongodb/MongoDbStorageDialect.java

```diff
@@ -65,11 +66,18 @@ public DataDocument addRecentlyUsedDocumentQuery(final String metadataCollection
             .append("update", new DataDocument()
                   .append("$push", new DataDocument()
                         .append(LumeerConst.Collection.RECENTLY_USED_DOCUMENTS_KEY, new DataDocument()
-                              .append("$each", Arrays.asList(id))
+                              .append("$each", Collections.singletonList(id))
                               .append("$position", 0)
                               .append("$slice", listSize))));
    }
 
+   public DataDocument[] usersOfGroupAggregate(final String organization, final String group) {
```

Comment:
This should be made more generic. Getting nested array will be needed in the other parts of the system as well. Moreover, we should not put domain-specific methods to the database storage implementation.